modules/explorer.go

```diff
@@ -5,89 +5,77 @@ import (
 )
 
 const (
+	// ExplorerDir is the name of the directory that is typically used for the
+	// explorer.
 	ExplorerDir = "explorer"
 )
 
-// Used for the BlockInfo call
 type (
-	ExplorerBlockData struct {
-		ID        types.BlockID   // The id hash of the block
-		Timestamp types.Timestamp // The timestamp on the block
-		Target    types.Target    // The target the block was mined for
-		Size      uint64          // The size in bytes of the marshalled block
-	}
+	// ExplorerStatistics returns a bunch of instantaneous statistics about the
+	// explorer at the current height.
+	ExplorerStatistics struct {
+		// General consensus information.
+		Height            types.BlockHeight
+		CurrentBlock      types.BlockID
+		Target            types.Target
+		Difficulty        types.Currency
+		MaturityTimestamp types.Timestamp
+		Circulation       types.Currency
```

Comment:
I assume this struct is identical to the one in the API package. You could embed it and therefore avoid the verbose conversion, or add it as an explicit field of the API struct.